tasks/gabrielsebrao/ListaDeContatos/app/src/main/res/layout/contact_list_activity.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/contact1"
```

Comment:
The contacts doesn't have a visible separator beetween them. Turn them closer to the prototype

Reply:
ok, thank you!